### ISSUE TITLE
Using gamescope with Dark Souls 2: Scholar of the First Sin results in a black screen

### ISSUE THREAD

**Issue by wantija** [ValveSoftware/gamescope#1572]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using any gamescope patches or a forked version of gamescope?

- [X] The issue occurs on upstream gamescope without any modifications

### Current Behavior

When launching Dark Souls 2: Scholar of the First Sin from Steam using gamescope causes the game to only render a black window , sounds from the game can still be heard.

### Steps To Reproduce

Launch Dark Souls 2: Scholar of the First Sin from Steam with the launch arguments "gamescope -- %command%"

### Hardware information

```markdown
- Distro: Arch Linux
- CPU: R7 5800X
- GPU: RX 6700XT
- Driver Version: Mesa 24.2.4
```

### Software information

```markdown
- Desktop environment: KDE Plasma 6.2
- Session type: Wayland
- Gamescope version: Mesa 24.2.4
- Gamescope launch command(s): see above
```

### Which gamescope backends have the issue you are reporting?

- [X] Wayland (default for nested gamescope)
- [X] DRM (default for embedded gamescope, i.e. gamescope-session)
- [ ] SDL
- [ ] OpenVR

**Comment by matte-schwartz** [comment]
I'm able to repro this on Arch with all backends. what's bizarre is that on SteamOS it seems to work completely fine for me with the DRM backend, so I would've expected the same on Arch while using the same Gamescope package. 

Separate from this, it seems to put my external display into a very weird colorspace with the DRM backend where mangohud and the Steam UI are super washed out. Disabling WSI fixes the colorspace issue but not the black screen.

I'm also on Mesa 24.2.4 right now so perhaps I should poke around there a bit for the black screen.

**Comment by matte-schwartz** [comment]
@wantija in `~/.local/share/Steam/steamapps/compatdata/335300/pfx/drive_c/users/steamuser/AppData/Roaming/DarkSoulsII/GraphicsConfig_SOFS.xml` try changing:
```
<Resolution-FullScreenWidth>1280</Resolution-FullScreenWidth>
<Resolution-FullScreenHeight>720</Resolution-FullScreenHeight>
```
to
```
<Resolution-FullScreenWidth>1920</Resolution-FullScreenWidth>
<Resolution-FullScreenHeight>1080</Resolution-FullScreenHeight>
```
for some reason this fixed it on my PC. on my SteamOS device it defaulted to 1920x1080p which is why I did not run into the issue.

**Comment by wantija** [comment]
Yep, that fixed it for me, thanks. Wonder if this is fixable with gamescope of if users will have to do this workaround.
